inbm/dispatcher-agent/tests/unit/sota/test_os_factory.py

```diff
@@ -33,16 +33,19 @@ def test_raise_error_unsupported_OsFactory(self) -> None:
         self.assertRaises(ValueError, factory.get_os, "MacOS")
 
     def test_create_ubuntu_snapshot_checker(self) -> None:
```

Comment:
Add unit tests to verify that the `create_snapshotter` method correctly handles the `reboot_device` parameter for each OS type. [important]

Reply:
These are tests to ensure the correct instance is created.  It has no bearing on whether that parameter is True or False.